.travis.yml

```diff
@@ -1,3 +1,4 @@
+arch: ppc64le
```

Comment:
This will make sure that our test suite run _only_ against PowerPC. Is that what we want?

If we want to run the test suite against several architectures the configuration should look like, e.g. ([related docs](https://config.travis-ci.com/ref/arch))

```yaml
arch:
- amd64
- arm64
- ppc64le
- s390x
```

We could in theory merge this, @jenisys, provided the test suite demonstrates to be passing. What would hold us back?

Reply:
@santosh653 A gentle nudge, you'd need to adapt your PR to get it merged. Please see the example above. ⬆️